helm/csi-powerstore/templates/node.yaml

```diff
@@ -136,6 +136,12 @@ spec:
               value: /powerstore-config-params/driver-config-params.yaml
             - name: GOPOWERSTORE_DEBUG
               value: "true"
+            {{- if hasKey .Values.node "healthMonitor" }}
+            {{- if eq .Values.node.healthMonitor.enabled true}}
```

Comment:
After `hasKey` check we can just set the value since it will be either true or false which avoids this additional check